components/Slides.vue

```diff
@@ -2,10 +2,12 @@
   <div class="flex relative box-border h-full reveal">
     <div class="slides flex h-full w-full">
       <section
-        :data-markdown="slidescontent"
+        data-markdown
         data-separator="^\r?\n---\r?\n$"
```

Comment:
```suggestion
        data-separator="^\r?\n===\r?\n$"
        data-separator-vertical="^\r?\n==\r?\n$"
```
@raar1 I have been thinking about this a bit and I actually suggest to right away change the slide delimiter to something else than the nuxt markdown front matter delimiter. I think this will make it slightly more user friendly. Even though we do not follow the reveal js default, we actually set it explicitly already. Also we can add the vertical slides at the same time. I think triple equal signs work well, and double equal signs for vertical slides.

Reply:
OK I agree - this will be safer than waiting for the inevitable "where did my first slide go?" incident...
But then I will update <https://github.com/esciencecenter-digital-skills/research-software-support/pull/82> to use the new separators